projects/core/styles/mixins/textfield.less

```diff
@@ -46,6 +46,11 @@
     .text-basic();
     .clearinput();
     .fullsize();
+
+    :host[data-mode='onDark'] & {
+        .clearinput(dark);
```

Comment:
This way you got all the same styles repeated twice, while only autofill should change. Probably should be `.autofill(dark)`

Reply:
replaced by `.autofill(dark)` ✅